lib/constants.js

```diff
@@ -55,7 +55,14 @@ const PEGOUT_EVENTS = {
   BATCH_PEGOUT_CREATED: "batch_pegout_created",
   PEGOUT_TRANSACTION_CREATED: "pegout_transaction_created",
   PEGOUT_CONFIRMED: "pegout_confirmed"
-}
+};
+
+const PEGIN_EVENTS = {
```

Comment:
Not for now, but I would consider splitting this file before it grows too much. 2wpConstants, BrideEvents, FedChangeConstants, etc